R/rows-dm.R

```diff
@@ -285,17 +285,27 @@ do_rows_append <- function(x, y, by = NULL, ..., in_place = FALSE, autoinc_col =
   source_rows <- map(key_values, ~ select(filter(y, !!returning == !!.x), -!!returning))
 
   con <- dbplyr::remote_con(x)
-  target_name <- dbplyr::remote_name(x)
-  insert_queries <- map(source_rows, ~ dbplyr::sql_query_append(
-    con,
-    target_name,
-    .x,
-    returning_cols = autoinc_col
-  ))
+  if (utils::packageVersion("dbplyr") >= "2.3.2.9000") {
+    insert_queries <- map(source_rows, ~ dbplyr::sql_query_append(
```

Comment:
`sql_query_append()` etc got an updated interface to make them more consistent to other `sql_query_*()` functions. In particular, it felt wrong to pass a lazy table to them.